invoices/invoiceregistry.go

```diff
@@ -198,12 +198,11 @@ func (i *InvoiceRegistry) populateExpiryWatcher() error {
 		return err
 	}
 
-	for idx := range pendingInvoices {
-		i.expiryWatcher.AddInvoice(
-			pendingInvoices[idx].PaymentHash, &pendingInvoices[idx].Invoice,
-		)
-	}
-
+	log.Debugf("Adding %v pending invoices to the expiry watcher")
+	t0 := time.Now()
+	i.expiryWatcher.AddInvoices(pendingInvoices)
+	log.Debugf("Added %v pending invoices to the expiry watcher, took: %v",
```

Comment:
Nit: I don't think this log is necessary, the one above gives enough context if things get stuck here for a while.

Reply:
Indeed, done.